test/serializers/attribute_test.rb

```diff
@@ -81,6 +81,23 @@ def id
         assert_equal('custom', hash[:blog][:id])
       end
 
+      def test_id_with_question_mark_has_question_mark_removed
+        serializer = Class.new(ActiveModel::Serializer) do
+          # TODO: if we were to remove the question mark automatically,
+          #       what do we do if we want to call a method that has a
+          #       question mark.
+          #       e.g.:
+          #               def my_question?; ....; end
```

Comment:
well, if that's the case, then I guess we should remove it, and just note in our docs, about the possibility of a collision, if someone were silly enough to try rendering both of `:boolean, :boolean?` where both of those point to different data.

maybe it's just an edge case, and it won't be that big of a deal.